.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
   - bash .travis/setup.sh
 
 install:
+  - pip install Cython
```

Comment:
Why not include in `requirements-test.txt`: https://github.com/h5py/h5py/issues/535#issuecomment-74833863